subsys/net/l2/openthread/Kconfig

```diff
@@ -257,6 +257,19 @@ config OPENTHREAD_SLAAC
 	help
 	  Enable SLAAC capability in OpenThread stack
 
+config OPENTHREAD_STACK_VENDOR_OUI
+	int "Vendor Organizationally Unique Identifier"
+	default 16043574
+	help
+	  Custom vendor OUI for OpenThread,
+	  which makes up 24 bits of MAC address
```

Comment:
How about `which makes 24 oldest bits of the MAC address`.

Reply:
Ok, done